service/src/main/kotlin/fi/espoo/evaka/document/childdocument/ChildDocumentQueriesCitizen.kt

```diff
@@ -29,6 +31,10 @@ fun Database.Read.getChildDocumentCitizenSummaries(
             )
         }
         .toList<ChildDocumentCitizenSummary>()
+        .filter {
+            VASU_MIGRATION_COMPLETED ||
```

Comment:
Also here I think `VASU_MIGRATION_COMPLETED` is unnecessary. And you could extract and reuse the filter implementation.